### PR TITLE
Add one-liner about using a VPN when working with restricted ISPs

### LINKED ISSUE #36472: Local installation fails to download the required dependencies
My ISP is CANTV Venezuela, which blocks Google and many other websites (government censorship). The "[Set up freeCodeCamp locally](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/docs/how-to-setup-freecodecamp-locally.md)" guide fails on the `npm run docker:install` and `npm ci` commands when I don't use a VPN. I don't know if fixing this is possible, but you could add something in the troubleshooting section about how ISP blocks can cause the install to get stuck on the "install dependencies" step with 0 warnings telling you that it's failing to download the files it needs, and telling people affected by this to try using a VPN or proxy.

### PR DESCRIPTION
**Pull request by Jo-IE**
- [ x] I have read [freeCodeCamp's contribution guidelines](https://github.com/freeCodeCamp/freeCodeCamp/blob/master/CONTRIBUTING.md).
- [ x] My pull request has a descriptive title (not a vague title like `Update index.md`)
- [x ] My pull request targets the `master` branch of freeCodeCamp.
- [ x] None of my changes are plagiarized from another source without proper attribution.
- [ x] All the files I changed are in the same world language (for example: only English changes, or only Chinese changes, etc.)
- [ x] My changes do not use shortened URLs or affiliate links.

Closes #36472

### CODE REVIEW

**Comment by thecodingaviator** [comment]
I don't see a reason why someone's ISP would block `docker` or `npm` in general except in case of uncleared dues or a government directive, as such, I believe that I speak for all of freeCodeCamp when I say that we would not like to promote anything that might be illegal.

cc: @raisedadead

File: docs/how-to-setup-freecodecamp-locally.md [hunk 1]
```diff
@@ -207,6 +207,8 @@ All of the above needs to be run only the first time you set up the local dev en
 npm ci
 ```
 
+**NB:** If your ISP has certain network restrictions, the above step (2) may fail without warning. In such a case, you will need to install a VPN or proxy to get this step to pass.
```
Comment (raisedadead) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion
If you get errors while installing the dependencies, please make sure that you are not in a restricted network or your firewall settings do not prevent you from accessing resources. Some solutions would be use to VPN services if possible and allowed in your environment.
```

Hi @Jo-IE 

Thanks for the PR. I think we should move this change to the troubleshooting area. Second we should update the language to be more neutral.

**Review by raisedadead — CHANGES_REQUESTED** [review]

**Comment by Jo-IE** [comment]
Thanks for the suggestion @raisedadead, I just committed it.

File: docs/how-to-setup-freecodecamp-locally.md [hunk 2]
```diff
@@ -207,6 +207,8 @@ All of the above needs to be run only the first time you set up the local dev en
 npm ci
 ```
 
+If you get errors while installing the dependencies, please make sure that you are not in a restricted network or your firewall settings do not prevent you from accessing resources. Some solutions would be use to VPN services if possible and allowed in your environment.
```
Comment (jonathan-grah) [inline docs/how-to-setup-freecodecamp-locally.md]:
```suggestion
If you get errors while installing the dependencies, please make sure that you are not in a restricted network or your firewall settings do not prevent you from accessing resources. One solution would be to use a VPN service if possible and allowed in your environment.
```

**Review by jonathan-grah — CHANGES_REQUESTED** [review]
Hi @Jo-IE, I also agree with @raisedadead that we should move this to the troubleshooting section (at the bottom of the page).

**Comment by Jo-IE** [comment]
Thanks for the clarifications, the latest commit should have addressed all the issues above.

**Comment by raisedadead** [comment]
Great the latest commit moved it to the correct location, can you update the verbiage as per https://github.com/freeCodeCamp/freeCodeCamp/pull/36485#discussion_r307085515

**Comment by Jo-IE** [comment]
Oh sorry I didn't catch that. They've been added now.

**Review by raisedadead — APPROVED** [review]

**Comment by raisedadead** [comment]
Thanks for the working on this @Jo-IE !

**Pull request merged**